tests/Unit/SimpleDTOTest.php

```diff
@@ -295,3 +296,22 @@ public function __invoke()
         ->and($user->email)
         ->toBe('john.doe@example.com');
 });
+
+it('custom casting with callback function', function () {
```

Comment:
Update description to `casts properties with castable classes and callables`

Reply:
Done - https://github.com/WendellAdriel/laravel-validated-dto/pull/38/commits/0909e089af2d1254ff3349c710e4bbb72ff0959d